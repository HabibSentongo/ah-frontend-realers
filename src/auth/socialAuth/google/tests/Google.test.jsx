import React from 'react';
// third-party libraries
import {
    shallow
} from 'enzyme';
// components
import GoogleButton from '../components/Google';
// Define a shared shallow renderer function

describe('Google Login', () => {
  // Pass the value of the shallow renderer before each test
  it('should render without errors', () => {
    const wrapper = shallow( <GoogleButton / > );
    expect(wrapper).toMatchSnapshot();
  });

  it("should test google button", () => {
    const componentWrapper = shallow(<GoogleButton />);
    const response = {
      access_token: {"Zi":'testToken'},
    };
    componentWrapper.instance().responseGoogle(response.access_token);
  });
});
