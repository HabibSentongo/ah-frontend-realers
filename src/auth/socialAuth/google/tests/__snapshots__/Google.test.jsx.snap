// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Google Login should render without errors 1`] = `ShallowWrapper {}`;
